tests/test_indexer.py

```diff
@@ -189,6 +189,8 @@ def _create_tombstone(self, tombstone_id):
         self.addCleanup(self._delete_tombstone, tombstone_id)
         sample_event = self.create_bundle_deleted_event(tombstone_id.to_key())
         self.process_new_indexable_object(sample_event, logger)
+        # Return the expected tombstone index document, which includes the bundle uuid.
+        tombstone_data['uuid'] = tombstone_id.uuid
```

Comment:
This feels a little out of place. Consider changing the assertion. The test should assert that tombstone_data is a subset of what's in the index and that it has the uuid property correctly set. We don't actually mind if there is more data in the index for a tombstone.

Reply:
This change was made. Please have a look at how the assert is being performed now.